app/controllers/drafts_controller.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'rest-client'
+require 'digest/md5'
+require 'aws-sdk-rails'
+require 's3_helper'
+
+class DraftsController < ApplicationController
```

Comment:
Spencer forwarded over a good PDF entitled Growing Rails Applications in Practice.

I'd recommend reading that and attempting to apply the Beautiful Controllers and Strong Parameters sections to this controller.